src/commands/app/undeploy.js

```diff
@@ -55,8 +56,11 @@ class Undeploy extends BaseCommand {
 
       for (let i = 0; i < keys.length; ++i) {
         const k = keys[i]
-        const v = values[i]
-        await this.undeployOneExt(k, v, flags, spinner)
+        let config = values[i]
+
+        config = setRuntimeApiHostAndAuthhandler(config)
```

Comment:
technically this modifies the argument (passed by reference), so you don't have to assign. if it returns a new cloned value, yes it is fine